bin/node2nix.js

```diff
@@ -57,7 +57,7 @@ var supplementNix = "supplement.nix";
 var nodeEnvNix = "node-env.nix";
 var lockJSON;
 var registries = [];
-var nodePackage = "nodejs-14_x";
+var nodePackage = "nodejs_14";
```

Comment:
```suggestion
var nodePackage = "nodejs_18";
```

Current node LTS version is v18, I'd prefer to use that as the default.

(I realize it might not have been the current LTS version at the time this PR was created.)